src/layouts/Authenticated.vue

```diff
@@ -34,7 +32,6 @@
               </div>
               <div class="flex flex-col ml-2 md:w-84 lg:w-84">
                 <form-select
-                  :key="currentIncidentId"
```

Comment:
Same here

Reply:
Sorry if it was added by intention. I only checked that there was no `v-for` being used in the context so decided to remove it  as it's useless. 
@tabiodun @BradenM plz let me know if it's needed to be added back. waiting for you guys.

Reply:
Yeah. In places where a key exists without a `v-for` it's deliberate to help refresh the component

Reply:
So these will be needed

Reply:
Ok, got it. I will add this back. Thank you for catching this out, @BradenM @tabiodun

Reply:
Now it's got key 🔙 . Please check.